Add CONTRIBUTING.md (#15)
DioxusLabs/stretch: CONTRIBUTING.md

# Contributing

This is a cross-team project, aiming to build solid foundations for Rust UI libraries of all sorts.
New contributions are extremely welcome!

The basic process is simple:

> 1. Pick an issue, or file a new one.

Reviewer: ```suggestion
1. Pick an [issue](https://github.com/DioxusLabs/stretch/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22), or [file a new one](https://github.com/DioxusLabs/stretch/issues/new).
```

2. Comment in the issue that you plan to tackle it, and the team will assign the task to you.
3. Submit a PR.
4. Respond to feedback from reviewers and make sure CI passes.

> PR review is completely open: if you'd like to help us comment on, test and approve PRs.

Author: ```suggestion
The PR review process is completely open:  help us by commenting on, testing and approving PRs.
```

> Questions of "how do I use this library" belong in GitHub Discussions, although those should typically spawn new issues to improve our docs and examples.

Reviewer: ```suggestion
Questions like "how do I use this library?" belong in [GitHub Discussions](https://github.com/DioxusLabs/stretch/discussions/new), although those should typically spawn new issues to improve our docs and examples.
```
